### DISCUSSION TITLE
Make the action only commit new, untracked files

### DISCUSSION THREAD

**Discussion by jj-lucas** [stefanzweifel/git-auto-commit-action#338] [Help] [answered]
I'm trying to use this action to only commit untracked files, rather than also modifications to existing ones.
Is this possible? How so?
Thanks

**Comment by stefanzweifel** [comment, marked as answer]
This isn't easily doable with git-auto-commit as - as far as I know – there is no option/flag for this built in into `git-status`, `git-add` or `git-commit`.

What you could do is run the following script in your action instead of using git-auto-commit.

```shell
git ls-files --others --exclude-standard | xargs git add
git commit -m "Add untracked files"
git push origin
```

You would have to add any missing git-commands yourself.

**Reply by jj-lucas** [reply]
thank you for looking into it :)
